Include Cloudfront changes for IRSA (#1522)
giantswarm/docs: src/content/advanced/iam-roles-for-service-accounts/index.md

 ...
```

With AWS release v18.0.0 we additionally need permissions for managing Cloudfront (**only non-China regions**). By upgrading to the release the S3 bucket including the S3 objects will be protected and become private, therefore we ensure only traffic through Cloudfront.

Please set the following permissons on the [`GiantSwarmAWSOperator` IAM role]({{< relref "/getting-started/cloud-provider-accounts/aws" >}}):

```json
...
  {
      "Effect": "Allow",
      "Action": [
          "cloudfront:TagResource",
          "cloudfront:UntagResource",
          "cloudfront:GetCloudFrontOriginAccessIdentity",
          "cloudfront:CreateCloudFrontOriginAccessIdentity",
          "cloudfront:DeleteCloudFrontOriginAccessIdentity",
          "cloudfront:GetDistribution",
          "cloudfront:CreateDistribution",
          "cloudfront:UpdateDistribution",
          "cloudfront:DeleteDistribution"
      ],
      "Resource": "*"

> },

Author: ```suggestion
},
...
```

 ...
```

> With AWS release v18.0.0 we additionally need permissions for managing Cloudfront (**only non-China regions**). By upgrading to the release the S3 bucket including the S3 objects will be protected and become private, therefore we ensure only traffic through Cloudfront.

Author: ```suggestion
With AWS release v18.0.0 we additionally need permissions for managing Cloudfront (**only non-China regions**). By upgrading to this AWS release the S3 bucket which contains the OIDC configuration and public keys will be protected and direct traffic will be blocked. We only allow traffic through Cloudfront.
```

### IAM role

> #### AWS Release v17.x.x

Author: ```suggestion
#### AWS Release v17.x.x or China regions
```

## Cross Account Roles

### IAM role cross account

In order to assume roles cross AWS accounts you will need to create a new AWS Identity Provider (OpenID Connect) in the AWS account where the IAM role is located.

![Creating AWS Identity Provider](identity-provider.png)

Login into the AWS on the account where the cluster is running:
- Grab the URL of the Identity Provider in your current cluster `IAM > Identity Providers`. It will look like `https://s3.REGION.amazonaws.com/AWS_ACCOUNT-g8s-CLUSTER_ID-oidc-pod-identity` or `https://CLOUDFRONT_ID.cloudfront.net`.

Login into the account where the IAM role is located and create an Identity Provider in `IAM > Identity Providers`:
- Set `Provider URL` to the previously gathered URL and click the `Get thumbprint` to import the certificate.
- Set the `audience` to `sts.amazonaws.com` OR `sts.amazonaws.com.cn` for China regions.

> #### AWS Release v17.x.x

Author: ```suggestion
#### AWS Release v17.x.x or China regions
```
